native/src/main/java/io/ballerina/stdlib/data/yaml/utils/JsonTraverse.java

```diff
@@ -0,0 +1,321 @@
+/*
```

Comment:
Don't we need to change this file name as `YamlTraverse`

Reply:
I guess no. Because we are traversing over a Json value. Any input @prakanth97?